mlir/test/Bytecode/general.mlir

```diff
@@ -36,3 +36,26 @@
   }) : () -> ()
   "bytecode.return"() : () -> ()
 }) : () -> ()
+
+
+// RUN: mlir-opt -allow-unregistered-dialect -emit-bytecode %s | mlir-opt -allow-unregistered-dialect | FileCheck %s
```

Comment:
This line is not needed, it'll run the test a second time, the line at the top is enough.

What can be done though is adding the `--split-input-file` to `mlir-opt` and add `// -----` separators between modules in the file.